index.js

```diff
@@ -795,14 +795,17 @@ function modifyAboutHome(track, step) {
             if (track === 'reward') {
                 contentURL = './tmpl/reward-snippet.html';
                 imageURL = imageBase + 'reward.png';
+                // load snippet HTML
+                var snippetContent = self.data.load(contentURL).replace('%url', self.data.url(imageURL));
```

Comment:
We can remove the `var snippetContent` from the inside of the if-else. So the code block will end up looking like:
```
 var snippetContent;
 if (track === 'reward') {
                contentURL = './tmpl/reward-snippet.html';
                imageURL = imageBase + 'reward.png';
                // load snippet HTML
                snippetContent = self.data.load(contentURL).replace('%url', self.data.url(imageURL));
            } else {
                // constructs uri to snippet content
                contentURL = './tmpl/' + track + '/content' + step + '-snippet.html';
                imageURL = imageBase + sidebarProps.track + '/content' + sidebarProps.step + '.gif';
                // load snippet HTML
                snippetContent = replaceSnippetCopy(track, contentURL, imageURL);
}
```